db/migrate/20160707095431_add_slug_to_grouped_issues.rb

```diff
@@ -0,0 +1,7 @@
+class AddSlugToGroupedIssues < ActiveRecord::Migration
+  def change
+    remove_column :grouped_issues, :uuid, :string
+    add_column :grouped_issues, :slug, :string
+    add_index :grouped_issues, :slug
```

Comment:
You forgot to add the uniq index for the slug column.. I'll add it here but you have to run `rake db:schema:load`!!